src/runtime/parallel/coordinator.ts

```diff
@@ -224,7 +224,7 @@ export default class Coordinator implements IRuntime {
       supportCodeLibrary: this.supportCodeLibrary,
     })
     return await new Promise<boolean>((resolve) => {
-      for (let i = 0; i <= this.numberOfWorkers; i++) {
+      for (let i = 0; i < this.numberOfWorkers; i++) {
```

Comment:
Without this change, we were actually starting up one more worker than the user wanter

ie if numberOfWorkers is 2, this loop was executing 3 times, i = 0,1,2